### DISCUSSION TITLE
[5.x]: Can't create product without variant

### DISCUSSION THREAD

**Discussion by grasmachien** [craftcms/commerce#3526] [General]
### Description
Since our client does not need variants they don't understand the interface to manage the products.
In Commerce 4 I was able to create a product without variants.
How can I enable this feature again in Commerce 5?
It was super easy to have all info in the details panel of the product page instead having to open the variant that does not actually exist at the bottom of the page.

### Steps to reproduce

1.  Go to the product type you want to edit
2. Set Max Variants to 0 => this gives no errors but does not save the value
3. Set Max Variants to 1 => this saves correctly
4. The products still show the layout for variants and I even see the button to add more variants even though I have set it to a maximum of 1 when there already is 1 variant.

### Expected behavior
There should only be 1 variant possible and the layout should be made easier in this case.

### Actual behavior
The user is super confused why he needs open another sidebar to edit the most crucial information of this product while it was accessible in the details panel in Commerce 4.

### Craft CMS version

5.1.7

### Craft Commerce version

5.0.7

### PHP version

8.2

### Installed plugins and versions

-

**Comment by linear** [comment]
<p><a href="https://linear.app/craftcms/issue/PT-1772/[5x]-cant-create-product-without-variant">PT-1772 [5.x]: Can't create product without variant</a></p>

**Comment by lukeholder** [comment]
Thanks for the feedback on the changes made in Commerce 5.

I can understand the client could be confused by this change and it does add extra UI steps. 

We will leave this discussion open for feedback from others, and will also discuss this internally to give further feedback on this request when we can.

Thanks

**Comment by mikejpeters** [comment]
We are in the same situation and would really appreciate a simpler UX for single variant products.

Ideally there should be a setting to inline the variant fields right onto the product form when there is only one, or if that's not possible, at least a simpler interface when "listing" the single variant (filters, multiple views, disabled buttons are all overkill in this situation).

Thanks!

**Comment by thisisjamessmith** [comment]
Running into this issue too migrating from Craft 4 - For product types that do not have multiple variants this is quite a UX regression for editors, involving lots more clicking, and lots more UI noise.
